Library.py

```diff
@@ -0,0 +1,131 @@
+import os
+class Library:
+
+    def __init__(self):
+        self.books = []
+        self.users = []
+        self.registry = []
+
+
+    def setup(self):
+        if os.path.exists("C:/Users/micha/Documents/Python/PytonMentoring/books.txt") and os.path.exists("C:/Users/micha/Documents/Python/PytonMentoring/users.txt"):
+            print("Library successful loaded! ")
+            self.books = open("books.txt").read().split()
+            # PRINT TESTOWY!!!
+            print(self.books)
+            self.users = open("users.txt").read().split()
+            # PRINT TESTOWY!!!
+            print(self.users)
+            self.registry = open("registry.txt").read().split()
+            # PRINT TESTOWY!!!
+            print(self.registry)
+
+        else:
+            print("Missing some database, Creating new one... ")
+            open("books.txt", "a")
+            open("users.txt", "a")
+            open("registry.txt", "a")
+            self.setup()
+
+    # sprawdz czy pliki books.txt i users.txt istnieją w folderze x
+    # jesli tak to zczytaj books i users do atrybutów
+    # jesli nie to komunikat, biblioteka pusta
+
+    def borrow(self, user, password, book):
+        if user in self.users and self.users[int(self.users.index(user) + 1)] == password:
+            if self.available_books(book):
+                self.registry.append(f"{user}={book}")
+                with open("registry.txt","a") as f:
+                    f.write(f"{user}={book}\n")
+                self.books.remove(book)
+                self.books.append(f"{book}Taken")
+                with open("books.txt", "r+") as f:
+                    file_data = f.read()
+                    file_data = file_data.replace(book, f"{book}Taken")
+                    with open("books.txt", "w") as f:
+                        f.write(file_data)
+                print(f"{user} just borrow book: {book}")
+        else:
+            print("Invalid inputs! ")
+
+    def deposit(self, user, password, book):
+        if user in self.users and self.users[int(self.users.index(user)+1)] == password and f"{user}={book}" in self.registry:
+            self.books.remove(f"{book}Taken")
+            self.books.append(book)
+            with open("books.txt", "r") as f:
+                file_data = f.read()
+                file_data = file_data.replace(f"{book}Taken", book)
+                with open("books.txt", "w") as f:
+                    f.write(file_data)
+            self.registry.remove(f"{user}={book}")
+            with open("registry.txt", "r") as f:
+                file_data = f.read()
+                file_data = file_data.replace(f"{user}={book}", "")
+                with open("registry.txt", "w") as f:
+                    f.write(file_data)
+            print(f"{user} just deposit book: {book}")
+        else:
+            print(f"{user} do not have book: {book}")
```

Comment:
To może również oznaczać złe hasło, dodaj taki case

Reply:
DONE